docs/deployments/changing-domains.mdx

```diff
@@ -1,5 +1,81 @@
 ---
-sanity_slug: /docs/deployments/change-production-domain
+title: Change domain or subdomain
+description: Learn how to change your production instance's domain or subdomain on Clerk.
 ---
 
-# This is a stub
+# Change domain or subdomain
+
+Learn how to change your production instance's domain or subdomain on Clerk.
+
+## Change domain
+
+Production domains can be changed in the [Clerk Dashboard](https://dashboard.clerk.com/) or via our [backend API](https://clerk.com/docs/reference/backend-api). Once you make the change to your domain, you will need to update the following:
+
+- Update the publishable key
+- Update DNS records
+- Generate new SSL certificates
+- If using social connections, update redirect URLs for enabled social connections
```

Comment:
Add: "You will also need to update the settings with your social connections, so the redirect URL they are using is correct."